packages/components/src/templates/dialog-samples.stories.ts

```diff
@@ -0,0 +1,129 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Dialog samples',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: ''
+    }
+  }
+};
+
+/**
+ * TODO: Add description and adjust story height.
+ *
+ * ```
+ * ```
+ */
+
+export const Default = {
+  name: 'Dialog with accordion',
+  render: () => html`
+    <style>
+      sd-dialog::part(footer) {
+        margin-left: 0;
+      }
+    </style>
+    <div class="min-h-[100vh]">
+      <sd-dialog id="dialog" open>
```

Comment:
In this use case the close button needs to be omitted because users have to click the primary button in order to close the dialog overlay.

Reply:
I’ve updated the content ... since Storybook is public we need to make sure we only show public information

[See Figma ](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2365-41319&t=2cNoGQ6ofKzqsyrT-11)

Reply:
How can we show the responsive behavior here?

Reply:
Viewport addon to display different viewports seems to not be available in the docs page. Did some research and could not find anything that we can plug n play to have it working. Need to discuss it with the team to check how we could approach it.